content/en/support.md update member figures, some language polishing

```diff
@@ -5,9 +5,9 @@ draft: false
 ---
 
 ## Who we are
-Our hack- and makespace, xHain, has existed for seven years. xHain is an open workshop and community for everyone interested in hacking and making, and has become an important meeting place for people in Berlin. We've been at Grünberger Straße 16 for four years.
-xHain is a non-profit organization that enables anyone to build projects, provides space for many important contemporary areas, such as discussing digital policy, and through the years has become an essential place for organising youth work.
-We currently have 130 members who support us. We organise Workshops for adults over a wide variety of topics that cover hacking and making. We cooperate with CoderDojo Germany e.V. and Jugend Hackt where young people can learn programming and other skills.
+Our hack- and makespace, xHain, has been around since 2018. xHain is an open workshop and community for everyone interested in hacking and making, and has become an important meeting place for people in Berlin.  
+xHain is a non-profit organization that enables anyone to build projects, provides space for many important contemporary areas, such as discussing digital politics. Over the years, it has become an essential place for organising youth activities.
+We currently have 200 members who support us. We organise Workshops for adults over a wide variety of topics that cover hacking and making. We cooperate with CoderDojo Germany e.V. and Jugend Hackt, where young people can learn programming and other skills.
```

Comment:
The single line breaks are misleading as they are not rendered as paragraphs. I suggest removing them or making actual paragraph breaks if that is the intention.

Reply:
Copy suggestion:

We have 200 members who support us. We organize workshops for adults on a wide variety of topics, including hacking and making. We partner with CoderDojo Germany e.V. and Jugend Hackt, where young people can learn programming and other skills.